person.rb

```diff
@@ -0,0 +1,25 @@
+# person.rb
+
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
```

Comment:
- In order to make your code more readable and maintainable, following the ruby style and convention, kindly move the private and the def of_age below the public method, doing so, would ensure that you don't explicitly indicate a method is private and then indicate another method is public, as this would be simplified if you have all your public method come first(not necessarily indicating they're public), then you use the private to indicate certain method are private. Your code should look like this below:

```ruby
  def can_use_services? 
    of_age?() || @parent_permission
  end

  private 

  def of_age?
    @age >= 18
  end
```